client/src/components/admin/AdminOverviewContainer.tsx

```diff
@@ -1,14 +1,15 @@
 import React, {Component} from "react";
 import {connect} from "react-redux";
-import {AppState} from "../../reducers/reducers";
-import AdminLinksCard, {AdminCardLink} from "./AdminLinksCard";
+import {AppState} from "../../state/Store";
+import AdminLinksCard from "./AdminLinksCard";
 import {Grid, Header} from "semantic-ui-react";
+import {AdminCardLink} from "../../types/Admin";
```

Comment:
why is this in types now?

Reply:
I'll move it back since it's only used in the one place